xm-commons-domainevent-db/src/main/java/com/icthh/xm/commons/domainevent/db/service/TypeKeyAwareEntityResolver.java

```diff
@@ -0,0 +1,24 @@
+package com.icthh.xm.commons.domainevent.db.service;
+
+import com.icthh.xm.commons.domainevent.db.domain.JpaEntityContext;
+import com.icthh.xm.commons.lep.AppendLepKeyResolver;
+import com.icthh.xm.lep.api.LepManagerService;
+import com.icthh.xm.lep.api.LepMethod;
+import com.icthh.xm.lep.api.commons.SeparatorSegmentedLepKey;
+import org.springframework.stereotype.Component;
+
+import static com.icthh.xm.commons.domainevent.db.service.impl.TypeKeyAwareJpaEntityMapper.TYPE_KEY;
+
+@Component
+public class TypeKeyAwareEntityResolver extends AppendLepKeyResolver {
```

Comment:
suggest remove to separate package for example `com.icthh.xm.commons.domainevent.db.lep`